src/Pay/Adapter/Stripe.php

```diff
@@ -0,0 +1,223 @@
+<?php
+
+namespace Utopia\Pay\Adapter;
+
+use Utopia\Pay\Adapter;
+
+class Stripe extends Adapter {
+
+    private string $baseUrl = 'https://api.stripe.com/v1';
+    private string $secretKey;
+    private string $publishableKey;
+
+
+    public function __construct(string $publishableKey, string $secretKey, string $currency = 'USD')
+    {
+        $this->secretKey = $secretKey;
+        $this->publishableKey = $publishableKey;
+        $this->currency = $currency;
+    }
+
+    /**
+     * Get name of the payment gateway
+     */
+    public function getName() : string {
```

Comment:
```suggestion
    public function getName(): string {
```

Reply:
Please ensure we comply with PSR-12 standards across the codebase 👍 

<img width="1220" alt="Screenshot 2022-04-17 at 12 22 59 PM" src="https://user-images.githubusercontent.com/20852629/163708517-9fc77d53-e4ae-4df4-a102-1c4099688a77.png">